server/command.go

```diff
@@ -406,6 +406,10 @@ func (p *Plugin) GetMSTeamsChannelDetailsForAllTeams(msTeamsTeamIDsVsChannelsQue
 }
 
 func (p *Plugin) executeConnectCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+	if storedToken, _ := p.store.GetTokenForMattermostUser(args.UserId); storedToken != nil {
```

Comment:
Just a question here. If this return error, in that storedToken will be nil and we are not checking the here. Should we allow the code flow to go any further ?